apps/android/app/src/main/java/chat/simplex/app/views/chat/ChatView.kt

```diff
@@ -118,11 +101,22 @@ fun ChatView(chatModel: ChatModel) {
           }
         }
       },
-      openDirectChat = { contactId ->
+      showChat = { apiId, chatType, pagination ->
```

Comment:
remove this coupling please - separate function that loads the initial chat (as it was) and add something like `loadMoreItem` that loads additional items on top - using API parameters (pagination) to manage view is not a good approach.